lib/pure/decimal.nim

```diff
@@ -0,0 +1,1040 @@
+##[
+  The ``decimal`` module supports the storage of numbers in decimal format.
```

Comment:
```suggestion
  The `decimal` module supports the storage of numbers in decimal format.
```
You can use single backticks now (which also do syntax highlighting by default afaik).

Reply:
fixed in next commit